src/poprox_concepts/domain/account.py

```diff
@@ -28,6 +28,7 @@ class AccountInterest(BaseModel):
     account_id: UUID | None = None
     entity_id: UUID
     entity_name: str
+    entity_type: str | None = None  # "subject", "person", "organisation", "place"
```

Comment:
We might want to make this either an enumeration or a string literal type in order to get validation of the type strings